src/methods/defining_methods/hard/q9/NumCoursesMethod.java

```diff
@@ -0,0 +1,26 @@
+package methods.defining_methods.hard.q9;
+
+/**
+ * [J] Write a Java Program that asks users for the number of courses they are taking right now. A course will contain
```

Comment:
Write a Java program asking the user for the number of courses they are taking right now. Then, create a method named creditsCac that accepts the number of courses as an integer and returns an integer for credits the student will receive, assuming that each course consists of three credits. Print the number of credits the user will receive in the main method.